protocol-definitions/DynamicConfig.yaml

```diff
@@ -903,6 +903,12 @@ methods:
           since: 2.5
           doc: |
             Tiered-Store configuration
+        - name: partitioningAttributeConfigs
+          type: List_PartitioningAttributeConfig
+          nullable: true
```

Comment:
just to double check, as most of the other configs here are not nullable: should we make this nullable?

Reply:
I'm not sure to be honest, IndexConfigs are optional and its a list too.